### PR TITLE
UI improvements for errors on migrations run and redo

### PR DESCRIPTION
**Pull request by sivakov512**
Hi!

Now output contains path to migration file if run/redo errored, like
```
Running migration 20170308040245
Executing migration script /path/to/migration/up.sql
Failed with : unrecognized token: "{"
```

refs #688

### CODE REVIEW

**Comment by sivakov512** [comment]
Can anybody help with this?
image
I dont know what it mean :(

**Comment by Dylan-DPC** [comment]
@sivakov512 it is an unrelated issue with clippy which strangely doesn't happen on other commits. Can you try restarting the build? (on travis, there is a button on the right).

**Comment by sivakov512** [comment]
No, i have no restart button, because i am not a project member.

**Comment by sgrif** [comment]
https://github.com/rust-lang-nursery/rust-clippy/issues/2910

**Comment by sivakov512** [comment]
@sgrif, I tried to update clippy version, but now there are [another problems](https://travis-ci.org/diesel-rs/diesel/jobs/402187177)

File: diesel/src/migration/errors.rs [hunk 1]
```diff
@@ -105,7 +105,7 @@ impl Error for RunMigrationsError {
 
 impl fmt::Display for RunMigrationsError {
     fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
-        self.description().fmt(f)
+        write!(f, "Failed with : {}", self.description())
```
Comment (sgrif) [inline diesel/src/migration/errors.rs]: We don't need a space before the colon

File: diesel/Cargo.toml [hunk 2]
```diff
@@ -15,7 +15,7 @@ categories = ["database"]
 byteorder = "1.0"
 diesel_derives = "~1.3.0"
 chrono = { version = "0.4", optional = true }
-clippy = { optional = true, version = "=0.0.195" }
+clippy = { optional = true, version = "=0.0.211" }
```
Comment (sgrif) [inline diesel/Cargo.toml]: Please avoid changes that are unrelated to your main PR. There's no reason to update clippy here.

**Comment by sivakov512** [comment]
@sgrif, fixed

File: diesel/src/migration/errors.rs [hunk 3]
```diff
@@ -105,7 +105,7 @@ impl Error for RunMigrationsError {
 
 impl fmt::Display for RunMigrationsError {
     fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
-        self.description().fmt(f)
+        write!(f, "Failed with:{}", self.description())
```
Comment (sgrif) [inline diesel/src/migration/errors.rs]: We still need a space after the colon

Comment (sivakov512) [reply diesel/src/migration/errors.rs]: Oh, sorry

**Comment by sivakov512** [comment]
I do not understand how to fix [the last problem in CI](https://travis-ci.org/diesel-rs/diesel/jobs/403119772). @sgrif, can you help me?

**Comment by sgrif** [comment]
It's been fixed on master, you need to rebase

**Comment by sivakov512** [comment]
@sgrif, I did something wrong and I had to create a new PR [here](https://github.com/diesel-rs/diesel/pull/1790).
